letsdrill.rb

```diff
@@ -1,12 +1,24 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+  case
+    when integer > 90
+      return "A"
+    when integer > 80
+      return "B"
+    when integer > 70
+      return "C"
+    when integer > 60
+      return "D"
+    else
+      return "F"
+  end
 
 end
 
 def shortest_string(array)
 
-  #Put your code here!
+  # thoughts on this syntax?
+  array.sort {|x,y| x.length <=> y.length }.first()
```

Comment:
Rubyists will strongly prefer `.first` to `.first()`.  There's also `sort_by`, and even `min_by`